### ISSUE TITLE
wick render image from last workflow when I drag in a new workflow

### ISSUE THREAD

**Issue by xueqing0622** [comfyanonymous/ComfyUI#2390]
wick render image display in other nodes from last workflow when I drag in a new workflow
![image](https://github.com/comfyanonymous/ComfyUI/assets/114840118/6ebf9420-dfec-4225-8363-a23a717fbe03)

**Comment by ltdrdata** [comment]
Yes, that's correct. It feels like it's been over a week since these symptoms started appearing. It seems that when loading the workflow, the generated image preview is moving to a node with the same Node ID.

I haven't tested it with vanilla ComfyUI, so I haven't been able to confirm whether it's specifically a ComfyUI issue or not.

**Comment by xueqing0622** [comment]
yes, I found it before use workspace manger:
https://github.com/11cafe/comfyui-workspace-manager/issues/56
![image](https://github.com/11cafe/comfyui-workspace-manager/assets/114840118/963be644-3c25-4e2a-a34d-203a9293b143)
![image](https://github.com/11cafe/comfyui-workspace-manager/assets/114840118/4fe89de5-3e30-4a1b-b478-d4862d08e72f)
I thought it was a problem with the workspace manger, but after deleting the workspace manger and still having the same problem, so I came to comfyui to report the error.

**Comment by abuyazid955** [comment]
i have the same problem, after i change to --lowvram, it's get back to normal, however the generation is so slow. i'm not using --lowvram before and it's works normally.

**Comment by xueqing0622** [comment]
why so long not fixed this issue
